src/OpenLoco/Widget.cpp

```diff
@@ -328,109 +327,109 @@ namespace OpenLoco::Ui
 
-    void Widget::draw_3(Gfx::Context* context, const Window* window, Widget* widget, uint16_t flags, uint8_t colour, bool enabled, bool disabled, bool activated)
+    void Widget::draw_3(Gfx::Context* context, const Window* window, uint16_t flags, uint8_t colour, bool enabled, bool disabled, bool activated)
     {
-        int16_t top, left, bottom, right;
-        top = window->y + widget->top;
-        left = window->x + widget->left;
-        right = window->x + widget->right;
-        bottom = window->y + widget->bottom;
+        int16_t t, l, b, r;
+        t = window->y + top;
+        l = window->x + left;
+        r = window->x + right;
+        b = window->y + bottom;
 
         if (activated)
         {
             flags |= 0x20;
         }
 
-        if (widget->content == -2)
+        if (content == -2)
         {
             flags |= 0x10;
-            Gfx::fillRectInset(context, left, top, right, bottom, colour, flags);
+            Gfx::fillRectInset(context, l, t, r, b, colour, flags);
             return;
         }
 
         if (window->flags & WindowFlags::flag_6)
         {
-            Gfx::fillRect(context, left, top, right, bottom, 0x2000000 | 52);
+            Gfx::fillRect(context, l, t, r, b, 0x2000000 | 52);
         }
 
-        Gfx::fillRectInset(context, left, top, right, bottom, colour, flags);
+        Gfx::fillRectInset(context, l, t, r, b, colour, flags);
 
-        if (widget->content == -1)
+        if (content == -1)
         {
             return;
         }
 
-        sub_4CADE8(context, window, widget, colour, enabled, disabled, activated);
+        sub_4CADE8(context, window, colour, enabled, disabled, activated);
     }
 
     // 0x004CABFE
-    void Widget::draw_5(Gfx::Context* context, const Window* window, Widget* widget, uint16_t flags, uint8_t colour, bool enabled, bool disabled, bool activated)
+    void Widget::draw_5(Gfx::Context* context, const Window* window, uint16_t flags, uint8_t colour, bool enabled, bool disabled, bool activated)
     {
-        if (widget->content == -1)
+        if (content == -1)
         {
             return;
         }
 
         if (!disabled)
         {
-            Widget::sub_4CADE8(context, window, widget, colour, enabled, disabled, activated);
+            sub_4CADE8(context, window, colour, enabled, disabled, activated);
             return;
         }
 
-        if (widget->type == WidgetType::wt_8)
+        if (type == WidgetType::wt_8)
         {
             return;
         }
 
-        if (widget->type != WidgetType::wt_7)
+        if (type != WidgetType::wt_7)
         {
-            Widget::sub_4CADE8(context, window, widget, colour, enabled, disabled, activated);
+            sub_4CADE8(context, window, colour, enabled, disabled, activated);
             return;
         }
 
-        // TODO: Remove image addition
-        uint32_t image = widget->image + 2;
+        // TODO: Remove added_image addition
+        uint32_t added_image = image + 2;
```

Comment:
Can you apply all the comments I had to the whole set of your changes not just the first ones.

Reply:
I am verry sorry, forgot some variables.
Now it shoud be fine.